src/main/java/uk/gov/companieshouse/pscfiling/api/controller/PscIndividualFilingController.java

```diff
@@ -28,7 +29,7 @@ public interface PscIndividualFilingController {
      * @throws NotImplementedException implementing classes must perform work
      */
     @PostMapping
-    default ResponseEntity<Object> createFiling(@PathVariable("transactionId") final String transId,
+    default ResponseEntity<PscCommunal> createFiling(@PathVariable("transactionId") final String transId,
```

Comment:
Is it possible to return `ResponseEntity<PscIndividualFiling>` instead?

Reply:
amended and re run code ok